README.md

```diff
@@ -47,12 +47,16 @@ curl -fsSL https://goss.rocks/install | GOSS_VER=v0.3.6 GOSS_DST=~/bin sh
 
 ```bash
 # See https://github.com/aelsabbahy/goss/releases for release versions
-curl -L https://github.com/aelsabbahy/goss/releases/download/_VERSION_/goss-linux-amd64 -o /usr/local/bin/goss
-chmod +rx /usr/local/bin/goss
+curl -L https://github.com/aelsabbahy/goss/releases/download/_VERSION_/goss-linux-amd64 -o ./goss-linux-amd64
```

Comment:
I would drop the documentation changes.

The installation instructions are intended to be quick/easy ones showing how simple goes is. If a user wants to take it further with checksums or installing somewhere other than /use/local/bin that's up to them, but I don't think that should be in the readme.

Reply:
👍